app/models/category.rb

```diff
@@ -6,5 +6,10 @@ class Category < ApplicationRecord
   validates :title, presence: true
   validates :description, presence: true,
   length: { maximum: 100 }
+  has_many :words, dependent: :destroy
+  
+  def feed
```

Comment:
No need for FEED method since there is  an easier way to display word for Elearning